sopel/modules/admin.py

```diff
@@ -153,6 +153,21 @@ def temporary_part(bot, trigger):
     _part(bot, channel, part_msg, save=False)
 
 
+@plugin.require_privmsg
+@plugin.require_admin
+@plugin.command('chanlist', 'channels')
+@plugin.priority('low')
+def channel_list(bot, trigger):
+    """Show channels Sopel is in."""
+    channels = list(bot.channels.keys())
+    channels.sort()
+    channels = ', '.join(channels)
```

Comment:
Same result, less steps:

```suggestion
    channels = ', '.join(sorted(bot.channels.keys()))
```

As a side note: if you remember, I wasn't bothered by re-assigning twice a value to a variable (it was `pattern`). However, there is something I didn't mention back then: it doesn't bother me when the variable stay of the same type. This is not the case here: first it's a list, then it's a string. That isn't a very good practice, and I tend to avoid that.

But heh it doesn't matter here, as you can do everything in one go.

Reply:
I'm really amused right now, because I started off with a one-liner and then split it into steps for clarity before making the PR. 😹